### ISSUE TITLE
Feature Request - [theoretical way to allow us to use more than one distro at a time]

### ISSUE THREAD

**Issue by anon817301** [CypherpunkArmory/UserLAnd#1246]
How to use this template:
Any line that begins with '#' is a **header**. Please leave the headers in as is, and fill in the information they suggest
below them. These templates are here to help us resolve your issue as quickly as possible. Leaving requested information out
of your issue will cause our issue template bot to request that your post be reformatted. If a header doesn't seem relevant
to your issue, either write 'N/A' or something similar below the header or consider using a different template.

# Describe the problem your feature request solves.
A clear and concise description of what the problem is. 

i love this app, i use it allot, but i want to use it more, allow us to choose the port ssh connects to, this will (theoretically) allow us to run diffrent distros at the same time, "but how will that help" you might ask, that is where lucky patcher/island come into play, you see, when you run userland ssh it connects to port 8022, so if you also want to use another userland it wont work because port 8022 is already in use so it will get stuck on a "starting service" and 90% of the time the working session will crash, so by using another port we *MIGHT* be able to run multiple userlands

tldr; cloning the app with lucky patcher and allowing us to choose the port will *THEORETICALLY* allow us to use multipe distros at the same time

# Describe the solution you'd like 

allow us to choose the port ssh uses

# Describe alternatives you've considered

Please add any other context or screenshots you think might help us resolve your issue.

out of context but i spent a good 3 hours trying to make ubuntu-gnome work, has anyone done it? if yes post your .vnc/xstartup
also arch gui doesnt work properly (only lxde works partially)

**Comment by corbinlc** [comment]
Closing as part of a repository cleanup. If this issue is still relevant, feel free to reopen it or file a fresh one.
